public/static/docs/command-reference/remote/modify.md

```diff
@@ -296,6 +296,28 @@ including obtaining the necessary credentials, and how to form `gdrive://` URLs.
   ```dvc
   $ dvc remote modify myremote credentialpath /path/to/my/creds/[FILE_NAME].json
   ```
+  
+  Google Cloud is migrating to allowing only Service Accounts for application logins. To ensure forward compatibility and better security - its best practice to set up a Goole Sevice account for your project. You can follow these steps:
```

Comment:
@vade could you please share a link with more information about this?

Reply:
If you dont use a service account, you'll get this warning in the console/terminal where you evoked DVC (or any other gcloud sdk using tool connecting with user auth rather than service account)

`UserWarning: Your application has authenticated using end user credentials from Google Cloud SDK. We recommend that most server applications use service accounts instead. If your application continues to use end user credentials from Cloud SDK, you might receive a "quota exceeded" or "API not enabled" error. For more information about service accounts, see https://cloud.google.com/docs/authentication/`

Reply:
@vade specifically about this part `is migrating to allowing only Service Accounts`

Reply:
I cant find the link where I read this was 'an eventual thing they would no longer allow' but, maybeI was dreaming.